cal-itp/benefits
# Add Prettier VSCode extension

## What this PR does
- Add the Prettier VS Code extension
- Set the Prettier extension to `format on save` for CSS, JS (.js, .json), but *not* HTML

## How to test this PR
- Run this container
- Save a JS (.js, .json) file and make sure it gets formatted (Example: Use single quotes instead of double quotes on a file and make sure it does get saved)
- Save a HTML file and make sure it does not get formatted (Example: Save the `alert-bar.html` file and make sure it does not get reformatted)
- Save a CSS file and make sure it does get formatted (Example: Put `h1, h2` on one line, and Prettier should put them on 2 separate lines)
- Test a Python file and make sure it does get properly formatted
